tests/collections/common/test_spc_tokenizer.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from unittest import TestCase
+
+import pytest
+
+from nemo.collections.common.tokenizers.sentencepiece_tokenizer import SentencePieceTokenizer
+from nemo.collections.common.tokenizers.tokenizer_utils import MODEL_SPECIAL_TOKENS
+
+
+class TestSPCTokenizer(TestCase):
```

Comment:
Is this test for sentence piece? How about having the name TestSentencePieceTokenizer or adding a comment?

Reply:
Is sentencepeiece tokenizer written by us?

Reply:
renamed, yes, this is the test for our implementation, taken from NeMo 0.11